src/main/java/org/spongepowered/api/entity/living/Living.java

```diff
@@ -287,4 +288,20 @@
      * @param invisible Whether this entity is invisible or not
      */
     void setInvisible(boolean invisible);
+
+    /**
+     * Returns whether this living entity is invisible to a given player.
+     *
+     * @param otherPlayer The player to toggle visibility of this entity for
+     * @return Whether this entity is invisible to the given player
+     */
+    boolean isInvisible(Player otherPlayer);
+
+    /**
+     * Sets whether this living entity is invisible to a given player.
+     *
+     * @param otherPlayer The player to toggle visibility of this entity for
+     * @param invisible Whether this entity is invisible to the given player
+     */
+    void setInvisible(Player otherPlayer, boolean invisible);
```

Comment:
Is this really in the scope of a `Living` entity?
I don't know where this functionality would go but I don't think it should be here.